src/webrtc/callFeed.ts

```diff
@@ -158,24 +130,31 @@ export class CallFeed extends TypedEventEmitter<CallFeedEvent, EventHandlerMap>
         return this._height;
     }
 
+    public get audioTrack(): CallTrack | undefined {
+        return this._tracks.find((track) => track.isAudio);
+    }
+
+    public get videoTrack(): CallTrack | undefined {
+        return this._tracks.find((track) => track.isVideo);
+    }
+
     private get hasAudioTrack(): boolean {
         return this.stream ? this.stream.getAudioTracks().length > 0 : false;
     }
 
-    private updateStream(oldStream?: MediaStream, newStream?: MediaStream): void {
+    protected get tracks(): CallTrack[] {
+        return [...this._tracks];
+    }
+
+    protected updateStream(oldStream?: MediaStream, newStream?: MediaStream): void {
```

Comment:
This method allows calling like `updateStream(oldStream)`
In this case `this._stream = undefined`.

I am confused that the method has two optional parameters. So her behavior is not very clear to me. The method can add, update or delete a stream.  Do you wont such a powerful method because to make it easier to handle add, update and delete more common? (Because I saw `public setNewStream()`)

Reply:
Since this is the way it's always been, I'd leave it alone for now and revisit it in another PR for the sake of fewer changed lines. Does that sound fine?

Reply:
fine